.github/workflows/test.yml

```diff
@@ -0,0 +1,13 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-16.04
```

Comment:
Is there any reason we're using 16.04 instead of 18.04?

Reply:
No good reason. I've updated!